x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_var_field.tsx

```diff
@@ -156,8 +156,25 @@ export const PackagePolicyInputVarField: React.FunctionComponent<{
             />
           );
         case 'select':
+          const selectOptions = options?.map((option) => ({ value, label: option.text }));
```

Comment:
it probably should be something like `value: option.value` here no?

Reply:
Oops, sorry Nicolas! It was working fine with
```
options?.map(({ value, text }) => ({ value, label: text }));
```
which I thought was nicer, but ESLint complained about it so I changed it to the above and I must have forgotten to check it. 🙈 Thanks, fixed.